packages/agents-core/src/db/schema.ts

```diff
@@ -679,6 +680,333 @@ export const credentialReferences = sqliteTable(
   ]
 );
 
+/**
+ * Dataset table
+ * 
+ * A collection of test cases/items used for evaluation. Contains dataset items
+ * that define input/output pairs for testing agents. Used for batch evaluation
+ * runs where conversations are created from dataset items. Each datasetRun
+ * specifies which agent to use when executing the dataset.
+ * 
+ * Includes: name, description, metadata (for tagging?? can also remove this), and timestamps
+ */
+export const dataset = sqliteTable(
```

Comment:
You will need to redo in postgres, since @miles-kt-inkeep merged his PR.

It's a small number of changes